README.md

```diff
@@ -1,36 +1,91 @@
-# Journal Policy Tracker Frontend
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
 
-## Available Scripts
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/codeisscience/journal-policy-tracker-frontend">
+    <img src="logo.png" alt="Logo">
+  </a>
 
-In the project directory, you can run:
+  <h1 style="text-align:center">Journal Policy Tracker Frontend</h1>
```

Comment:
No biggie for now, but in the readme let's try to use markdown unless we really need HTML to keep maintenance simpler.

Reply:
True, I'll sort out the HTML issue in the next commit